src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -84,23 +85,20 @@ public static void setUp() throws IOException, InterruptedException, CreateApiKe
         // Mock the OTP server TODO: Run a live OTP instance?
         TestUtils.mockOtpServer();
         // As a pre-condition, create an API User with API key.
-        apiUser = PersistenceUtil.createApiUser(String.format("test-%s@example.com", UUID.randomUUID().toString()));
+        apiUser = createApiUser(String.format("test-%s@example.com", UUID.randomUUID().toString()));
         apiUser.createApiKey(DEFAULT_USAGE_PLAN_ID, true);
+        Persistence.apiUsers.replace(apiUser.id, apiUser);
```

Comment:
Shouldn't this happen in `createApiKey` if `persist=true`?

Reply:
yep, updated.